src/IT/it.test.js

```diff
@@ -33,7 +33,7 @@ test('Testing router to call calculator', async () => {
             return 'Mock return from calculator'
     })
 
-    const response = await request(app).get('/test/calculator');
+    const response = await request(app).get('/test/calculator').query({ operation: "addition", valueOne: 1, valueTwo: 2 });
```

Comment:
@preetham-b-capillary I am using ,supertest' for IT, and trying to mock calculator(in TC-3 ), but not able to do it here.
The TC is failing, could you make the correction here..